catroid/src/androidTest/java/org/catrobat/catroid/Localization/BricksCategoriesTest.java

```diff
@@ -0,0 +1,179 @@
+        BrickCategoryMotion =Espresso.onView(withText(R.string.category_motion));
+        BrickCategoryLooks =Espresso.onView(withText(R.string.category_looks));
+        BrickCategoryData =Espresso.onView(withText(R.string.category_data));
+        BrickCategoryPen =Espresso.onView(withText(R.string.category_pen));
+
+    }
+
+    @Test
+    public void assertNoEllipsizedTextInRTLMode()
+    {
+        BrickCategoryControl.check(noEllipsizedText());
+        BrickCategorySound.check(noEllipsizedText());
+        BrickCategoryMotion.check(noEllipsizedText());
+        BrickCategoryLooks.check(noEllipsizedText());
+        BrickCategoryData.check(noEllipsizedText());
+        BrickCategoryPen.check(noEllipsizedText());
+
+    }
+
+    @Test
+    public void assertNotNullValueInRTLMode()
+    {
+        BrickCategoryControl.check(matches(notNullValue()));
+        BrickCategorySound.check(matches(notNullValue()));
+        BrickCategoryMotion.check(matches(notNullValue()));
+        BrickCategoryLooks.check(matches(notNullValue()));
+        BrickCategoryData.check(matches(notNullValue()));
+        BrickCategoryPen.check(matches(notNullValue()));
+
+    }
+
+    @Test
+    public void assertNoOverLappingBricksInRTLMode()
+    {
+        BrickCategoryControl.check(noOverlaps());
+        BrickCategorySound.check(noOverlaps());
+        BrickCategoryMotion.check(noOverlaps());
+        BrickCategoryLooks.check(noOverlaps());
+        BrickCategoryData.check(noOverlaps());
+        BrickCategoryPen.check(noOverlaps());
+
+    }
+
+    @Test
+    public void assertLayoutDirectionIsRTL()
+    {
+        BrickCategoryControl.check(isLayoutDirectionRTL());
+        BrickCategorySound.check(isLayoutDirectionRTL());
+        BrickCategoryMotion.check(isLayoutDirectionRTL());
+        BrickCategoryLooks.check(isLayoutDirectionRTL());
+        BrickCategoryData.check(isLayoutDirectionRTL());
+        BrickCategoryPen.check(isLayoutDirectionRTL());
+
+    }
+
+    @Test
+    public void assertTextDirectionIsRTL()
+    {
+        BrickCategoryControl.check(isRTLTextDirection());
+        BrickCategorySound.check(isRTLTextDirection());
+        BrickCategoryMotion.check(isRTLTextDirection());
+        BrickCategoryLooks.check(isRTLTextDirection());
+        BrickCategoryData.check(isRTLTextDirection());
+        BrickCategoryPen.check(isRTLTextDirection());
+
+    }
+
+    @Test
+    public void assertIsVisibleBrickInRTL()
+    {
+        BrickCategoryControl.check(isVisible());
+        BrickCategorySound.check(isVisible());
+        BrickCategoryMotion.check(isVisible());
+        BrickCategoryLooks.check(isVisible());
+        BrickCategoryData.check(isVisible());
+        BrickCategoryPen.check(isVisible());
+
+    }
+
+    private void navigateProject()
+    {
+        Espresso.onView(withId(R.id.main_menu_button_continue)).perform(ViewActions.click());
+        Espresso.onView(withText("كائن1")).perform(ViewActions.click());
```

Comment:
can you please put all those button names and expected strings into static final string variables with english names (e.g. BUTTON_NAME_PROGRAMS) at the top of this test class, so developers who cannot read these languages can understand what this test does.